bindata/v3.11.0/openshift-controller-manager/ds.yaml

```diff
@@ -45,6 +45,8 @@ spec:
           name: client-ca
         - mountPath: /var/run/secrets/serving-cert
           name: serving-cert
+        - mountPath: /etc/pki/ca-trust/extracted/pem
```

Comment:
This duplicates how we map the same config map into build pods a la https://github.com/openshift/openshift-controller-manager/blob/master/pkg/build/controller/strategy/util.go#L549-L578

Reply:
Cross referencing my https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/145#issuecomment-601771069 here ... the installer does not appear to like mounting directly to `/etc/pki/ca-trust/extracted/pem`

Reply:
that error indicates to me that something else is already adding a mount to that path to the DS.  Not understanding why you think it's related to the installer though.  That error is:

1) installer complains that the controller manager opeator is reporting degraded
2) controller manager operator reported degraded because it's operand (daemonset) is not deployed successfully
3) daemonset is not deployed successfuly because k8s did not like that it had duplicate mount entries for the same path:  invalid: spec.template.spec.containers[0].volumeMounts[4].mountPath: Invalid value: \"/etc/pki/ca-trust/extracted/pem\": must be unique\nWorkloadDegraded:

So the next step is to figure out if something else in the operator code is adding the same mount path to the daemonset.

Reply:
sorry my "installer does not apper to like .." was my refraining from spelling out all the detail you just listed and just trying to short hand it somehow

bottom line, yes, need to figure out what is up ... I concur your next step is on point